plugins/json-multisheet-extractor/src/parser.spec.ts

```diff
@@ -0,0 +1,170 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { parseBuffer } from './parser'
+
+describe('parser', function () {
+  const buffer: Buffer = fs.readFileSync(
+    path.join(__dirname, '../ref/test-basic.json')
+  )
+
+  test('JSON to WorkbookCapture', () => {
```

Comment:
Suggestions on keeping things DRY with respect to the repeated headers in this file https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1826003564